.github/actions/run-cypress-tests/action.yml

```diff
@@ -80,13 +80,13 @@ runs:
       run: |
         cd ./OpenSearch-Dashboards
         echo "Start checking OpenSearch Dashboards."
-        for i in {1..60}; do
+        for i in {1..90}; do
           if grep -q "bundles compiled successfully after" "dashboard.log"; then
             echo "OpenSearch Dashboards compiled successfully."
             break
           fi
-          if [ $i -eq 60 ]; then
-            echo "Timeout for 600 seconds reached. OpenSearch Dashboards did not finish compiling."
+          if [ $i -eq 90 ]; then
+            echo "Timeout for 900 seconds reached. OpenSearch Dashboards did not finish compiling."
```

Comment:
Do you think we should file an issue with OSD Core repo about the increase in startup latencies witnessed in this repo?

Reply:
I think we are facing the same issue, that the compilation is taking extremely longer than before

Reply:
Yes, separated out to compile time step